recipes/tadbit/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --force
```

Comment:
Since it’s just a single line, you can move this into the `meta.yaml` (as `script:` under the `build:` section). Also, won’t the standard `"{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"` work here?

Reply:
I see now that you this is the first recipe you’re contributing, so I’m trying to find where this is written in the Bioconda  documentation. It seems the docs now recommend using `conda skeleton pypi`, which producese a `meta.yaml` without the `--no-deps` etc. part. Previously, the docs would the line I wrote above, and many recipes use i, so I don’t know what the curently recommend approach is. I’ll have to defer to someone who knows more.

Reply:
We didn't go to pip, just directly to conda recipe. Is that a problem?

Reply:
Maybe I should go for pip first ...

Reply:
Ignore the stuff about conda skeleton. It doesn’t matter how you created the recipe. Just remove the `build.sh` file and add this to your build section:
```
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

Reply:
I think we need to update the docs to include `{{ PYTHON }} -m pip install . --no-deps -vv` at least. I'll add that to the core agenda to add that as a point of discussion.

Reply:
Then I'll need to change my setup.py. We originally had a raw input for the user to confirm installation even if a dependency is not present. That blocks the bioconda install so I include a --force argument to bypass that.
But with pip I'll have to change the way the argument --force is passed to the setup.py.

Reply:
Just add `mcl` to your `host:` requirements. Then it’ll be found at `pip install` time.

(But I would also recommend reworking your `setup.py`, it looks quite complicated.)

Reply:
Main problem is for IMP in the salilab conda channel. That cannot be added to the host.
I'll work on my setup.py ... tonight